run/index.md

```diff
@@ -160,7 +160,7 @@ Today, of course, we'll be talking about [**Nextflow** :octicons-link-external-2
 
 Nextflow is a workflow language that is based on the Groovy programming language, which itself is based on Java. Its design philosophy is based on the concept of **dataflow programming**, which is a programming paradigm that models a program as a directed graph of the data flowing between operations. In Nextflow, these operations are called **processes** and the data flowing between them are called **channels**. 
 
-When you run Nextflow, you create a head job that manages the workflow and submits jobs to the local machine or a job scheduler. Each process is run in its own isolated environment, which can be a docker or singularity container, a conda environment, or just the local environment. This means that each process can have its own dependencies and versions of software without interfering with other processes.
+When you run Nextflow, you create a head job that manages the workflow and submits jobs to the local machine or a job scheduler (like SLURM). Each process is run in its own isolated environment, which can be a docker or singularity container, a conda environment, or just the local environment. This means that each process can have its own dependencies and versions of software without interfering with other processes.
 
 <!-- Need image diagram of head job submitting child jobs-->
```

Comment:
Also, is there a way for nextflow to generate an image of the rulegraph? could be good for the example workflows. If not, how does one get a sense for what a workflow does?